### PR TITLE
docs: update PWT page [sc-13036]

### PR DESCRIPTION
**Pull request by miliberlin**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer

Preview at `docs/browser-checks/playwright-test`. Anything else that should be added?

### CODE REVIEW

File: site/content/docs/browser-checks/playwright-test.md [hunk 1]
```diff
@@ -7,28 +7,53 @@ menu:
     parent: "Browser checks"
 ---
 
-Checkly natively supports running browser checks using Playwright Test Runner, allowing you to write tests and use assertions using the popular testing framework.
+Checkly natively supports running browser checks using Playwright Test Runner, allowing you to write tests and use assertions using the popular testing framework. 
+Read more on how to usitilise Playwright Test best in the [official documentation](https://playwright.dev/docs/writing-tests).
```
Comment (pilimartinez) [inline site/content/docs/browser-checks/playwright-test.md:11]: Is usitilise a typo from utilise?

**Review by pilimartinez — CHANGES_REQUESTED** [review]
Just a small typo, the rest looks good.

**Review by pilimartinez — DISMISSED** [review]
LGTM

File: site/content/docs/browser-checks/playwright-test.md [hunk 2]
```diff
@@ -63,29 +94,69 @@ This is the list of Playwright Test Runner features that are currently supported
...
 
 Here's an example of a Playwright Test script that fails, and provides a video of the test sequence.
 
-![checkly-pw-videos](https://user-images.githubusercontent.com/3121310/195104104-79f1abd8-ac85-4453-968f-e9afa3d8f314.gif)
+![checkly-pw-videos](/docs/images/browser-checks/pwt_videos.gif)
 
-> Note: When you use the editor page, for your convenience, 
-> we will always make the videos available for download and preview, also for passing tests.
+> **Note:** When running tests from the editor page, video files are always available for download and preview, 
+> regardless of whether the check is passing or failing. For scheduled checks files are only preserved when the check failed.
```
Comment (jan-osch) [inline site/content/docs/browser-checks/playwright-test.md:162]:
```suggestion
> regardless of whether the check is passing or failing. For scheduled check runs videos and traces are only preserved when the check failed.
```

File: site/content/docs/browser-checks/playwright-test.md [hunk 3]
```diff
@@ -63,29 +94,69 @@ This is the list of Playwright Test Runner features that are currently supported
 | Test retry               | No           |
 | Parallelism and sharding | No           |
```
Comment (jan-osch) [inline site/content/docs/browser-checks/playwright-test.md:96]: In the table above: can you mention that TS can be enabled via labs w/ link?

**Review by jan-osch — DISMISSED** [review]
❤️

**Review by jan-osch — APPROVED** [review]
Looks great!

**Pull request merged**